libs/components/text-editor/src/lib/modules/text-editor/text-editor.component.spec.ts

```diff
@@ -463,13 +508,78 @@ describe('Text editor', () => {
       validateToolbarActions(testComponent.toolbarActions);
     });
 
+    [
+      {
+        desc: 'new window',
+        windowOption: 'new',
+        urlStrings: [
+          'href="https://google.com',
+          'rel="noopener noreferrer"',
+          'target="_blank"',
+        ],
+        containsHintTextExpect: (
+          inputBoxText = document.querySelector('.sky-modal .sky-input-box')
```

Comment:
An input box can have more text content than just the hint text. You can select one level deeper with `.sky-input-box-hint-text`. I'd also recommend adding a specific CSS class to the input box you're selecting (like `sky-text-editor-url-input`) so that adding another input box before this one in the future doesn't break this test. Something like this:

```
inputBoxText = document.querySelector('.sky-modal .sky-text-editor-url-input .sky-input-box-hint-text');
```

Reply:
Done.